code/modules/events/holiday/xmas.dm

```diff
@@ -46,12 +46,20 @@
 	var/christmas_tree = /obj/structure/flora/tree/pine/xmas/presents
 
 /obj/effect/landmark/xmastree/Initialize(mapload)
-	..()
+	. = ..()
+	// Xmas tree landmarks on Centcom will initialize BEFORE SSevent is initialized
```

Comment:
This definitely doesn't make much sense, all atoms should call initialize at the same time when the atoms subsystem starts up, and then after that they call it immediately after new.

Reply:
Nevermind, they're marked as INITALIZE IMMEDIATE

Reply:
landmerks are dumb